packages/frontend/src/components/common/ResourceList/ResourceTable.tsx

```diff
@@ -99,6 +99,22 @@ const ResourceTable: FC<ResourceTableProps> = ({
             : new Map(),
     );
 
+    const getResourceUrl = (resource: AcceptedResources) => {
+        const resourceType = getResourceType(resource);
+
+        switch (resourceType) {
+            case 'dashboard':
+                return `/projects/${projectUuid}/dashboards/${resource.uuid}/view`;
+            case 'chart':
+                return `/projects/${projectUuid}/saved/${resource.uuid}`;
+            default:
+                return assertUnreachable(
+                    resourceType,
+                    `Can't get URL for ${resourceType}`,
+                );
+        }
+    };
```

Comment:
We could move this outside of the component, so we don't need to add it to the hook dependencies